packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -30,7 +36,15 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
         padding: 0,
         textAlign: 'left',
         textOverflow: 'inherit',
-        userSelect: 'text'
+        userSelect: 'text',
```

Comment:
Can we consider, in addition to this, putting a transparent bottom outline/border on it (or around the whole thing)? For FF support? @betrue-final-final

Reply:
I think previously we discussed adding a border bottom in hc mode, but the border bottom wouldn't be in the same place as the underline so we decided against it. As an example, in the image below, the first line is the underline, and the second line is the border. @betrue-final-final can confirm for this instance.
![image](https://user-images.githubusercontent.com/16619843/42114739-04f0d80e-7ba5-11e8-92c9-7828ed86898d.png)

Reply:
Philip, are you saying that in FF HC mode a border would appear?

Reply:
Oh jeeze I totally forgot we had a conversation about this. My stance was that I felt it wasn't important if the border was a bit further away than the underline, having something is still better than having nothing. Either that or we could put a box around the whole thing in HC so at least there's some indication of interactibility in FF. Since we've already committed to supporting FF accessibility where the cost is cheap enough. (And we could even turn the border off in IE HC.)

@betrue-final-final I'm saying that we COULD set 'border-bottom: 1px solid transparent;' or 'border: 1px solid transparent' and it would show up in HC in FF and IE, or if we wanted we could scope it to only FF.

Reply:
Anyways as usual I'll defer to Ben's decision on this.

Reply:
If we could scope this to just FF that would be best, and border-bottom only.

Reply:
@phkuo what's a good selector for FF? I've tried a couple from the interwebs and none have worked.

Reply:
@lynamemi Oh, in this case I would just put the border-bottom in the original style, and then in the IE-only HC selector overwrite it to be border-bottom:none.